modules/custom-status/custom-status.php

```diff
@@ -582,25 +606,28 @@ public function add_custom_status( array $args ): WP_Term|WP_Error {
 	 * @param array $args Any arguments to be updated
 	 * @return object $updated_status Newly updated status object
 	 */
-	public function update_custom_status( int $status_id, array $args = [] ): WP_Term|WP_Error {
-		$old_status = $this->get_custom_status_by( 'id', $status_id );
+	public static function update_custom_status( int $status_id, array $args = [] ): WP_Term|WP_Error {
+		$old_status = self::get_custom_status_by( 'id', $status_id );
 		if ( is_wp_error( $old_status ) ) {
 			return $old_status;
 		} else if ( ! $old_status ) {
 			return new WP_Error( 'invalid', __( "Custom status doesn't exist.", 'vip-workflow' ) );
 		}
 
 		// Reset our internal object cache
-		$this->custom_statuses_cache = [];
+		self::$custom_statuses_cache = [];
+
+		// Don't allow changing the name or slug of the Draft or pending status as they are core statuses
+		$banned_statuses = [ 'draft', 'pending' ];
```

Comment:
As with [the prior comment](https://github.com/Automattic/vip-workflow-plugin/pull/59#discussion_r1792538988), I'd love it if we could at least make "pending" deleteable.

Reply:
https://github.com/Automattic/vip-workflow-plugin/pull/59#discussion_r1792705172